spec/proteus_spec.rb

```diff
@@ -4,24 +4,38 @@
   before(:all) do
     remove_dummy_repo
     @proteus = Proteus::Kit.new
+    @repos = Proteus::REPOS
   end
```

Comment:
These methods aren't expensive, so I'd just do `before do end` here so that they execute on every spec.